docs/guide/http_cache.md

```diff
@@ -72,573 +80,753 @@ fos_http_cache:
+    ```
+
+    When trusting remote IP like this, make sure your application is only accessible through Varnish.
+    If it is accessible in other ways you may end up trusting e.g. IP of client browser instead which would be a serious security issue.
+
+    Make sure that **all** traffic always comes from the trusted proxy/load-balancer,
+    and there is no other way to configure it.
+
+
+See [Examples for configuring eZ Platform](#Examples-for-configuring-eZ-Platform) for how these variables can be set.
+
+
+#### Update YML configuration
+
+Secondly, you need to tell eZ Platform to use an HTTP-based purge client (specifically the FosHttpCache Varnish purge client),
+and specify the URL Varnish can be reached on (in `config/packages/ezplatform.yaml`):
+
+
+| Configuration | Parameter| Environment variable| Possible values|
+|---------|--------|--------|----------|
+| ezplatform.http_cache.purge_type | `purge_type` | `HTTPCACHE_PURGE_TYPE` | local, varnish, fastly |
+| ezplatform.system.(scope).http_cache.purge_servers | `purge_server`* | `HTTPCACHE_PURGE_SERVER`* | Array of URLs to proxies when using Varnish or Fastly (`https://api.fastly.com`) |
+| ezplatform.system.(scope).http_cache.varnish_invalidate_token | `varnish_invalidate_token` | `HTTPCACHE_VARNISH_INVALIDATE_TOKEN` | (Optional) For token based authentication |
+| ezplatform.system.(scope).http_cache.fastly.service_id | `fastly_service_id` | `FASTLY_SERVICE_ID` | Service ID to authenticate with Fastly |
+| ezplatform.system.(scope).http_cache.fastly.key | `fastly_key` | `FASTLY_KEY` | Service key/token to authenticate with Fastly |
+
+_\* If you need to set multiple purge servers configure them in the YAML configuration, instead of parameter or environment variable as they only take single string value._
+
+Example for configuring varnish as reverse proxy, [assuming front controller has been configured](#Configuring-Symfony-Front-Controller):
 
 ``` yaml
 ezplatform:
     http_cache:
         purge_type: varnish
 
     system:
-        my_siteacess:
+        # Assuming that my_siteaccess_group contains both your front-end and back-end SiteAccesses
+        my_siteaccess_group:
             http_cache:
-                purge_servers: ['http://varnish.server1', 'http://varnish.server2', 'http://varnish.server3']
+                # Fill in your Varnish server(s) address(es).
+                purge_servers: [http://my.varnish.server:8081]
 ```
 
-!!! tip "Environment Variables"
+!!! note "Invalidating Varnish cache using tokens"
 
-    eZ Platform uses environment variables by default, so you can define those values in the environment.
-    See [Update your Virtual Host](#update-your-virtual-host)
+    In setups where the Varnish server IP can change (for example on platform.sh/eZ Platform Cloud),
+    you can use token-based cache invalidation via [ez_purge_acl.](https://github.com/ezsystems/ezplatform-http-cache/blob/v2.1.0/docs/varnish/vcl/varnish5.vcl#L174)
```

Comment:
hmm.. We should likely document how to configure purge token too..

Reply:
Indeed

Reply:
Indeed, but unless you have proposal on that I suggest that should be followup.